tests/metrics/density/docker_memory_usage.sh.in

```diff
@@ -22,43 +22,89 @@
 #  This test launches certain number of containers and puts them into
 #  sleep mode for a certain period of time. Then the test checks the
 #  amount of memory used by each container
-#  This test uses psstop tool to get the memory used.
+#  This test uses smem tool to get the memory used.
 
 set -e
 
-[ $# -ne 2 ] && ( echo >&2 "Usage: $0 <times to run> <wait time>"; exit 1 )
+[ $# -ne 2 ] && ( echo >&2 "Usage: $0 <cc number> <wait time>"; exit 1 )
 
 SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
 source "${SCRIPT_PATH}/../../lib/test-common.bash"
 
 CMD='sh'
-IMAGE='ubuntu'
-TIMES="$1"
+IMAGE='busybox'
```

Comment:
("because it's small"?) ---> yeah, maybe @grahamwhaley  can give you more details about that :)